docker/README.md

```diff
@@ -5,17 +5,21 @@ EXECUTION_MODE: Indicates the execution mode of the module. It can take three va
     - local: It will execute locally, either using a local spark context or a spark cluster. (Not needed to setup, it is the default value)
     - mesos: It will execute using a mesos cluster.
     - yarn: It will execute using a yarn cluster.
-SPARK_HOME: Only needed if EXECUTION_MODE is 'mesos' or 'yarn'. Indicates the path of the spark compilation for hadoop.
-MESOS_MASTER: Only needed if EXECUTION_MODE is 'mesos'. Indicates the ip or fqdn of the Mesos master host.
+MESOS_MASTER: Only needed if EXECUTION_MODE is 'mesos'. Indicates the ip or fqdn of the Mesos master host (i.e. mesos.demo.stratio.com:7077).
 HDFS_MASTER: Only needed if EXECUTION_MODE is 'mesos' or 'yarn'. Indicates the ip or fqdn of the Yarn master host.
+HDFS_PORT: Only needed if EXECUTION_MODE is 'mesos' or 'yarn'. Indicates the port of the Yarn master host.
+SPARK_VERSION: Only needed if EXECUTION_MODE is 'mesos', 'yarn' or 'standalone'. Indicates the spark version.
+HADOOP_SPARK_VERSION: Only needed if EXECUTION_MODE is 'mesos', 'yarn' or 'standalone'. Indicates the compiled hadoop spark version.
+HADOOP_VERSION: Only needed if EXECUTION_MODE is 'mesos', 'yarn' or 'standalone'. Indicates the hadoop version.
+SPARK_MASTER: Only needed if EXECUTION_MODE is 'standalone'. Indicates the ip or fqdn of the Spark master host (i.e. spark.demo.stratio.com:7077).
 
 Usage examples
 ===============
 - Local execution:
-docker run -dit --name sp --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com:5000/stratio/sparta:0.9.0
+docker run -dit --name sp --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com/stratio/sparta:0.11.0
   
 - Mesos execution:
-docker run -dit --name sp --env EXECUTION_MODE=mesos --env SPARK_HOME=/spark-1.5.2-bin-hadoop2.6 --env MESOS_MASTER=mm.demo.stratio.com --env HDFS_MASTER=hm.demo.stratio.com --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com:5000/stratio/sparta:0.9.0
+docker run -dit --name sp --env EXECUTION_MODE=mesos --env MESOS_MASTER=mm.demo.stratio.com --env HDFS_MASTER=hm.demo.stratio.com --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com/stratio/sparta:0.11.0
   
 - Yarn execution:
-docker run -dit --name sp --env EXECUTION_MODE=yarn --env SPARK_HOME=/spark-1.5.2-bin-hadoop2.6 --volumes-from hs --env HDFS_MASTER=hm.demo.stratio.com --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com:5000/stratio/sparta:0.9.0
+docker run -dit --name sp --env EXECUTION_MODE=yarn --env HDFS_MASTER=hm.demo.stratio.com --env ZOOKEEPER_HOST=zk.demo.stratio.com qa.stratio.com/stratio/sparta:0.11.0
```

Comment:
qa.stratio.com isn't a public service. Images should not include any host, so are downloaded from hub.docker.com